### PR TITLE
add test for paging with duplicate filters

### PR DESCRIPTION
**Pull request by iscai-msft**

### CODE REVIEW

File: swagger/paging.json [hunk 1]
```diff
@@ -173,7 +218,10 @@
     },
     "/paging/multiple/nextOperationWithQueryParams": {
       "get": {
-        "x-ms-pageable": { "nextLinkName": null, "itemName": "values" },
+        "x-ms-pageable": {
+          "nextLinkName": "null",
```
Comment (weidongxu-microsoft) [inline swagger/paging.json:222]: @iscai-msft 

This is line wrong? Should it be `"nextLinkName": null` (no quotes)?

Comment (iscai-msft) [reply swagger/paging.json:222]: oh yeah that does seem wrong. It's showing up in the diff because I reformatted the swagger, so this code has been wrong for a long time. Let me do a separate PR to update this test
